application-form/src/components/admin/AdminHome.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <div class="admin-page">
+    <SideBar @handleLogout="handleLogout" :user="userDetail" />
+    <div class="main-content">
+      <div class="table-container">
+        <table class="table-item" cellspacing="0">
+          <tr class="table-header">
+            <th v-for="(item, index) in header" :key="index">
+              <p>{{ item }}</p>
+            </th>
+          </tr>
+          <UserList
```

Comment:
đặt tên là list nhưng lại v-for ra, component này chứa mỗi thông tin user thôi mà nhỉ